eggdrop/content.md

```diff
@@ -0,0 +1,37 @@
+# What is Eggdrop?
+
+Eggdrop is the world's most popular Open Source IRC bot, designed for flexibility and ease of use, and is freely distributable under the GNU General Public License (GPL). It is designed to Linux, BSD, SunOs, Windows, and Mac OS X, among others. The core codebase is extendable via TCL scripts or C modules and bots can be linked to form botnets, enabling the sharing of userfiles and partylines across multiple bots.
+
+%%LOGO%%
+
+# How to use this image
+
+## First Run
+
+To run this container the first time, you'll need to pass in, at minimum, a nickname and server via Environmental Variables. At minimum, a docker run command similar to
+
+	docker run -ti -e NICK=FooBot -e SERVER=irc.freenode.net eggheads/eggdrop:latest
```

Comment:
`s!eggheads/eggdrop:latest!eggdrop:latest!`

Might want to mention that `-i` is required even when running with `-d` on the `docker run`, since the process will exit if there is no `stdin`.  I am not sure if `-t` is required or useful as well on a background eggdrop container.